orchestra/proc-macro/src/impl_subsystem_ctx_sender.rs

```diff
@@ -72,6 +72,18 @@ pub(crate) fn impl_subsystem_types_all(info: &OrchestraInfo) -> Result<TokenStre
 					}
 				}
 			}
+
+			impl ::std::convert::From< #all_messages_wrapper > for #outgoing_wrapper {
+				fn from(message: #all_messages_wrapper) -> Self {
+					match message {
+					#(
+						#all_messages_wrapper :: #subsystem_generic ( msg ) => #outgoing_wrapper :: #outgoing_variant ( msg ),
+					)*
+						#all_messages_wrapper :: Empty => #outgoing_wrapper :: Empty,
```

Comment:
Looks dangerous. At the very least should be `TryFrom` and the caller should guarantee the invariant.